molecule/default/inventory/group_vars/all.yml

```diff
@@ -42,6 +42,10 @@ clickhouse_user_files:
   - users.d/01-first_test_user.yml
   - file: users.d/02-second_test_user.yml
     no_log: true
+  - file: users.d/03-third_test_user.yml
+    # TODO: properly test that attributes are supported
```

Comment:
Tests for attributes does not work now, but OK, they were never worked so I will merge it as-is

If you will be interested - https://github.com/semrush/ansible-role-clickhouse/issues/34 (this will be very welcome!)